website/docs/zh/guide/optimization/profile.mdx

```diff
@@ -2,6 +2,40 @@ import { ApiMeta, Stability } from '@components/ApiMeta.tsx';
 
 # 性能分析
 
+本章节介绍 Rspack 中一些常见的性能瓶颈和性能分析方法。
+
+## 性能瓶颈
+
+尽管 Rspack 本身提供的构建性能已经足够优秀，但在 Rspack 中使用一些 JavaScript 编写的 loaders 和 plugins 时，它们可能会导致整体的构建性能劣化，尤其是在大型项目中。
+
+对于这类问题，有一部分可以通过 Rspack 内置的高性能替代方案来解决，另一部分可以通过性能分析工具来定位，从而采取针对性的优化。
```

Comment:
add link to profile page  https://rspack.dev/guide/optimization/profile ?

Reply:
This is the profile page